network/p2pNetwork_test.go

```diff
@@ -1040,6 +1051,7 @@ func TestP2PWantTXGossip(t *testing.T) {
 	net.wantTXGossip.Store(false)
 	net.nodeInfo = &nopeNodeInfo{}
 	net.config.ForceFetchTransactions = true
+	net.config.NetAddress = "127.0.0.1:0"
```

Comment:
this should not be needed, it checks `ForceFetchTransactions`

Reply:
you were right, modifying